cross/rutorrent/Makefile

```diff
@@ -20,6 +20,8 @@ include ../../mk/spksrc.cross-cc.mk
 
 .PHONY: myInstall
 myInstall:
+	mv $(WORK_DIR)/ruTorrent-master $(WORK_DIR)/$(PKG_NAME)
```

Comment:
Please change PKG_DIR to `ruTorrent-master`, and drop this line. You'll need to make other changes accordingly. 
Setting EXTRACT_DIR might also work.